bot/exts/valentines/valentine_zodiac.py

```diff
@@ -19,36 +22,116 @@ class ValentineZodiac(commands.Cog):
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
-        self.zodiacs = self.load_json()
+        self.zodiacs, self.zodiac_fact = self.load_comp_json()
 
     @staticmethod
-    def load_json() -> dict:
+    def load_comp_json() -> Tuple[dict, dict]:
         """Load zodiac compatibility from static JSON resource."""
-        p = Path("bot/resources/valentines/zodiac_compatibility.json")
-        with p.open(encoding="utf8") as json_data:
+        explanation_file = Path("bot/resources/valentines/zodiac_explanation.json")
+        compatibility_file = Path("bot/resources/valentines/zodiac_compatibility.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            zodiac_fact = load(json_data)
+            for _, zodiac_data in zodiac_fact.items():
+                zodiac_data['start_at'] = datetime.fromisoformat(zodiac_data['start_at'])
+                zodiac_data['end_at'] = datetime.fromisoformat(zodiac_data['end_at'])
```

Comment:
use values() instead of items()
```suggestion
            for zodiac_data in zodiac_fact.values():
                zodiac_data['start_at'] = datetime.fromisoformat(zodiac_data['start_at'])
                zodiac_data['end_at'] = datetime.fromisoformat(zodiac_data['end_at'])
```